source/dem/particle_wall_contact_force.cc

```diff
@@ -49,16 +50,34 @@ ParticleWallContactForce<dim>::update_contact_information(
   particle_angular_velocity[2] =
     particle_properties[DEM::PropertiesIndex::omega_z];
 
+  // Calculate approximation of the contact point using the normal vector
+  Point<3> contact_point =
+    particle_position +
+    0.5 * particle_properties[DEM::PropertiesIndex::dp] * normal_vector;
+
+  // Get vector pointing from the contact point to the axis
+  Tensor<1, 3> vector_to_rotating_axis =
+    contact_point - this->point_on_rotation_vector[boundary_id];
+
+  // Remove the rotating axis component of that vector
+  vector_to_rotating_axis =
+    vector_to_rotating_axis -
+    (vector_to_rotating_axis * this->boundary_rotational_vector[boundary_id]) *
+      this->boundary_rotational_vector[boundary_id] /
```

Comment:
this->boundary_rotational_vector is not already a unit vector?

Reply:
we are using the `double boundary_rotational_speed`, so technically `boundary_rotational_vector `should be a unit vector. We should add a step when creating that container to make sure that we are storing unit vectors in it. This would save the extra division (and addition at the denominator). 

Also, why are you using the norm_square?

Reply:
I am using the norm square because we do two scalar product. There is no garantee right now that the vector is a unit vector, (e.g. someone could write 1,1,1 and it should still work). We can refactor this, but this means changing what was the meaning of these parameters. I am all for this, but this should be done in another PR. This PR fixes the problem we had, it does not aim at refactoring the entire parameter structure.